packages/image-block/src/ImageBlock.spec.ct.tsx

```diff
@@ -81,6 +82,84 @@ describe('Image Block', () => {
         cy.get(DownloadSelector).should('not.exist');
     });
 
+    it('should render as a button if the custom security settings allow it even if the global is disabled', () => {
+        const [ImageBlockWithStubs] = withAppBridgeBlockStubs(ImageBlock, {
+            editorState: false,
+            blockSettings: {
+                security: Security.Custom,
+                assetViewerEnabled: true,
+            },
+            privacySettings: {
+                assetViewerEnabled: false,
+                assetDownloadEnabled: true,
+            },
+            blockAssets: {
+                [IMAGE_ID]: [AssetDummy.with(1)],
+            },
+        });
+        mount(<ImageBlockWithStubs />);
+        cy.get(ImageBlockImageSelector).should('exist');
```

Comment:
Usesless to check if the ImageBlockImageSelector exists same for the other tests

Reply:
Not useless:

1. Without it the "not.exists" tests are false positive, because cypress runs the tests too fast, and asserts the not.exists without even waiting for the dom to be properly rendered.
2. It makes sure that the img tag itself is rendered.

Reply:
Ah I see, then its fine :D